src/main/java/com/exadel/discountwebapp/user/mapper/UserMapper.java

```diff
@@ -3,27 +3,27 @@
 import com.exadel.discountwebapp.user.entity.User;
 import com.exadel.discountwebapp.user.vo.UserRequestVO;
 import com.exadel.discountwebapp.user.vo.UserResponseVO;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.convention.MatchingStrategies;
 import org.springframework.stereotype.Component;
 
+import javax.annotation.PostConstruct;
+
 @Component
 public class UserMapper {
 
-    public static User toEntity(UserRequestVO requestVO){
-        return User.builder()
-                .firstName(requestVO.getFirstName())
-                .lastName(requestVO.getLastName())
-                .email(requestVO.getEmail())
-                .password(requestVO.getPassword())
-                .build();
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    @PostConstruct
+    public void init(){
+        modelMapper.getConfiguration().setMatchingStrategy(MatchingStrategies.STRICT);
+    }
+
+    public User toEntity(UserRequestVO requestVO){
+        return modelMapper.map(requestVO, User.class);
```

Comment:
it seems like we should also link the user with a role during the creation (by passing roleId in the user request)

Reply:
due to requirements we don't have opportunity to create or update(for now) users in db, so this method is unnecessary. deleted